src/components/calendar-view/CalendarView.tsx

```diff
@@ -94,7 +95,7 @@ function Component({
         selectedDate: selectedDate,
         selectedRangeStart: selectedRangeStart,
         selectedMultiDates: selectedMultiDates,
-        viewDate: viewDate ? fromString(viewDate) : undefined,
+        viewDate: viewDate,
```

Comment:
Changed to string so that during re-render the props compare does not return true if the date object is for the same date value.

**Scenario**: If `viewDate` is updated after press of a `shortcutButton` and the user changes the month, we need to pass a new `viewDate` object into the props when the `shortcutButton` is pressed again (since a string `viewDate` type value remains the same and comp will not re-render).